Files/Helpers/GlyphHelper.cs

```diff
@@ -1,103 +0,0 @@
-                {
-                    iconCode = "\uEB9F";
-                }
-                else if (path.Equals(udp.Music, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = "\uEC4F";
-                }
-                else if (path.Equals(udp.Videos, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = "\uE8B2";
-                }
-                else if (path.Equals(AppSettings.NetworkFolderPath, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = "\uE8CE";
-                }
-                else if (Path.GetPathRoot(path).Equals(path, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = "\uEDA2";
-                }
-            }
-            return iconCode;
-        }
-
-        public static Uri GetIconUri(string path)
-        {
-            Uri iconCode = new Uri("ms-appx:///Assets/FluentIcons/Folder.svg");
-            if (path != null)
-            {
-                // TODO: do library check based on the library file path?
-                var udp = UserDataPaths.GetDefault();
-                if (path.Equals(AppSettings.DesktopPath, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Desktop.svg");
-                }
-                else if (path.Equals(AppSettings.DownloadsPath, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Downloads.svg");
-                }
-                else if (path.Equals(udp.Documents, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Documents.svg");
-                }
-                else if (path.Equals(udp.Pictures, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Pictures.svg");
-                }
-                else if (path.Equals(udp.Music, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Music.svg");
-                }
-                else if (path.Equals(udp.Videos, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Videos.svg");
-                }
-                else if (path.Equals(AppSettings.NetworkFolderPath, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Drive_Network.svg");
-                }
-                else if (Path.GetPathRoot(path).Equals(path, StringComparison.OrdinalIgnoreCase))
-                {
-                    iconCode = new Uri("ms-appx:///Assets/FluentIcons/Drive_USB.svg");
-                }
-            }
-            return iconCode;
-        }
-    }
-}
```

Comment:
Can you restore this helper? It's still something we can use for areas like the tab control.

Reply:
@yaichenbaum Doesn't seem to be used at all in my testing, otherwise the app wouldn't have compiled successfully.

Reply:
@duke7553 It's not currently used, but it's something to do down the line. What was the compile error?

Reply:
@yaichenbaum There was no compiler error. And something like this, we can just rewrite.

Reply:
@duke7553 if the plan is to use it, why would we remove it?